### PR TITLE
Fix a mistake in Secrets concept

### PR DESCRIPTION
**Pull request by luoxiaohi**

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
Please tell why this is a mistake in this PR's desc. Do not leave it blank.

**Comment by luoxiaohi** [comment]
> Please tell why this is a mistake in this PR's desc. Do not leave it blank.

this section is talking about **the `kubernetes.io/tls` Secret type**, but its content mistakenly refers to `kubernetes.io/ssh-auth`, which is described in the previous section.

**Comment by sftim** [comment]
@luoxiaohi would you be willing to edit the existing description for this PR? That will help reviewers.

File: content/en/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -381,7 +381,7 @@ The following YAML contains an example config for a TLS Secret:
 
 The TLS Secret type is provided only for convenience.
 You can create an `Opaque` type for credentials used for TLS authentication.
-However, using the defined and public Secret type (`kubernetes.io/ssh-auth`)
+However, using the defined and public Secret type (`kubernetes.io/tls`)
```
Comment (kbhawkey) [inline content/en/docs/concepts/configuration/secret.md:384]: Hello @luoxiaohi .
Do you have any more information on this change? It's not clear that the `kubernetes.io/tls` Secret type is correct.

**Comment by sftim** [comment]
/retitle Fix a mistake in Secrets concept

File: content/en/docs/concepts/configuration/secret.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/configuration/secret.md:384]: This matches line 364.

**Review by sftim — APPROVED** [review]
Thanks

/lgtm
/approve

**Pull request merged**